1906-06-11.xml

```diff
@@ -1375,8 +1376,1163 @@
+                <div type="section" scope="local">
+                    <head>THE KHEDIVE</head>
+                    <div type= "item" scope="local"> 
+                        <head>ARRIVAL IN CONSTANTINOPLE.</head>
+                        <p>CONSTANTINOPLE, June 9.</p>
+                        <p>H.H. the Khedive has arrived. (Havas)</p> 
+                        <p>CONSTANTINOPLE, June 10.</p>
+                        <p>The Khedive arrived yesterday and lunched at the Palace. He afterwards
+                            drove to his residence on the Bosphorus. (Reuter)</p>
+                    </div>
+                    <div type= "item" scope="local"> 
+                        <head>RHEDIVIAL DECORATIONS.</head>
+                        <p>H.H. the Khedive has conferred grades and decorations on several
+                            officials who have retired on pension. Mabmoud Sofwat Bey, director of
+                            the Tanzim, Gharbieh, receives the rank of Montemaiz. The Sanieh is
+                            conferred on Mohamed Hanafi Bey, chief of the Arabic Department of the
+                            State Railway Administration, Cairo ; and the same grade on Khorshid
+                            Wessim Bey, inspector of Balkas Teftiche, State Domains. Constantin Eff.
+                            Hanna, employe in the Government Printing Offioe, has received the
+                            Medjidieh of the 5th class.</p>                        
+                    </div>
+                    <div type= "item" scope="local"> 
+                        <head>KHEDIVIAL DECREES.</head>
+                        <p>Saturday's "Offioial Journal” contains the following decrees :—</p>
+                        <p>1. A decree raising the pay of 2nd lieutenants in the Egyptian Army to
+                            L.E. 6 per mensem (L.E. 8 in the Sudan) and of 1st lieutenants to L.E. 7
+                            per mensem (L.E. 9 in the Sudan).</p>
+                        <p>2. A decree authorising the construction of two bridges over the Sayalah
+                            Draw at El Khohak and another over the El Ramadi Canal at El Sebaieh, in
+                            the Assouan province.</p>
+                        <p>3. A decree authorising the construction of a railway signal box at
+                            Miniet el Sireg, Kaliou- bieh.</p>
+                        <p>4. Authorisation of the enlargement of the left embankment of the El
+                            Rashwanieh Canal, Ghirgeh province.</p>
+                        <p>5. Authorisation of the construction of a new station at Aga, on the
+                            light railway between Mit Ghamr and Mansoorah.</p>
+                        <p>6. Authorisation of the expropriation of land at Tamalai for a new Moslem
+                            cemetery.</p>
+                    </div>
+                    <div type= "item" scope="local"> 
```

Comment:
add a `</div>` to close the khedive section and to make the following item independent of it.